src/__tests__/analytics-test.ts

```diff
@@ -368,16 +387,19 @@ test('logEvent in web is resilient to gtag script not working', async () => {
         }
     };
 
-    await expect(logEvent({name: 'any_event'})).resolves.toBeUndefined();
+    await expect(logEvent({ name: 'any_event' })).resolves.toBeUndefined();
```

Comment:
please, revert formatting changes. Probably you had a bad prettier configuration in your editor, or you forgot to run `yarn` to install the prettier config. CI job is failing because of these changes

Reply:
Yes, I fighting with my Junior AI Developer. It started pretty well but is tired right now. We will keep updating the PR next week.